massa-ledger/src/ledger.rs

```diff
@@ -7,8 +7,8 @@ use crate::ledger_entry::LedgerEntry;
 use crate::types::{Applicable, SetUpdateOrDelete};
 use crate::{FinalLedgerBootstrapState, LedgerConfig, LedgerError};
 use massa_hash::hash::Hash;
-use massa_models::{Address, Amount, Slot};
-use std::collections::{BTreeMap, VecDeque};
+use massa_models::{Address, Amount};
+use std::collections::BTreeMap;
 
 /// Represents a final ledger associating addresses to their balances, bytecode and data.
 /// The final ledger is also attached to a final slot, can be boostrapped and allows others to bootstrap.
```

Comment:
I think it'd be nice to specify that the ledger is contained in state and that state is what is attached to a final slot to avoid misconceptions.

Reply:
corrected